apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/styles/starter-page-templates-editor.scss

```diff
@@ -406,7 +406,7 @@ body.admin-bar:not( .is-fullscreen-mode ) .page-template-modal-screen-overlay {
 			> .editor-inner-blocks
 			> .editor-block-list__layout
 			> [data-type='core/column']
-			> .editor-block-list__block-edit
```

Comment:
Same as above (let's keep the old `.editor-` selectors and move these changes to a separate PR).

Reply:
Hmmm I wonder if this is actually something we should be concerned at all here. I can see several other occurrences of the non-legacy/newer formats in this and other files in the plugin. I feel a more concentrated effort ought to be carried out for this type of thing? There's at least 20+ occurrences of `block-editor-block` suffixed classes in the plugin that could potentially be candidates for legacy-support. 🤔

Reply:
I'm not familiar with what's the backwards compatibility policy we have on the FSE plugin, but I think we tried to have it compatible with at least the 2 latest releases of WP Core (so FSE can work even if the Gutenberg plugin is not active). So I'd assume that any existing selector is currently working with WP 5.2 and we shouldn't worry about them. @Automattic/cylon or @Automattic/ajax might know more.

Reply:
Ok. My argument is just that if `block-editor-block__x` won't work here due to missing support for older browsers, then it won't work anywhere. I am not sure if it makes a lot of sense to add support here and leave it hanging loose elsewhere. I feel a more holistic effort would be appropriate.

Reply:
I'd defer to @noahtallen for a final call on this, but personally I don't think we'd need to be too worried about WP back compat. The plugin is meant to work with simple and atomic, third-party users are installing it on their own risk. See https://github.com/Automattic/wp-calypso/blob/master/apps/full-site-editing/full-site-editing-plugin/readme.txt#L32

Reply:
+1 for what @obenland said. We require the latest version of the Gutenberg dev plugin in a lot of cases, so we are fine updating everything to the most recent APIs as long as Atomic / Simple sites also have that. 

I think we removed the previous 2 WP version requirement a few months back. I agree with what @chriskmnds is saying about some of the syntax. We've been trying to keep up with newer CSS selectors as we've noticed things breaking, so we've also removed older classnames as well when we've made those updates.

The only case I can think of is covering the gap between 
- FSE plugin update is released with new classnames
- Gutenberg plugin update is released to Simple sites
- Gutenberg plugin update is released to Atomic sites.

Things could break in the period between when one of those is released and all of them is released.

Reply:
Thanks for the input folks. I believe we are aligned then. 🙂

p.s. it should be safe to apply these refactors to FSE at any point. It's just legacy selectors replaced with the new ones. I will prepare a PR.

Reply:
https://github.com/Automattic/wp-calypso/pull/39035